api/src/main/resources/workbench.yaml

```diff
@@ -1099,6 +1099,38 @@ paths:
           description: The cohort definition that was created.
           schema:
             $ref: "#/definitions/Cohort"
+        400:
+          description: 'Bad Request. Cohort name already exists'
+
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/cohorts/duplicate:
+    parameters:
+      - $ref: '#/parameters/workspaceNamespace'
+      - $ref: '#/parameters/workspaceId'
+    post:
+      tags:
+        - cohorts
+      description: Creates a duplicate of the cohort specified by cohortId
+      operationId: "duplicateCohort"
+      parameters:
+        - in: body
+          name: DuplicateCohortParams
```

Comment:
This is an interesting pattern. What does the JSON look like here? The convention we have elsewhere is to define a request object, but I think we should probably switch if this is equivalent (if we want to do this pattern, please also file a ticket on this).

Reply:
I changed it to look more like what we have elsewhere. Which I believe is... `[Method]Request` defined in definitions and making a reference to it.

The definition of the object has remained the same.